integrations/ScriptLoader.js

```diff
@@ -9,10 +10,16 @@ const ScriptLoader = (id, src, async = defaultAsyncState) => {
   js.async = async === undefined ? defaultAsyncState : async;
   js.type = "text/javascript";
   js.id = id;
-  const e = document.getElementsByTagName("script")[0];
-  logger.debug("==parent script==", e);
-  logger.debug("==adding script==", js);
-  e.parentNode.insertBefore(js, e);
+  const headElems = document.getElementsByTagName("head");
+  if (Object.keys(headElems).length !== 0) {
+    logger.debug("==adding script==", js);
+    headElems.insertBefore(js, headElems.firstChild);
```

Comment:
Would this work if there is no first child?

Reply:
yes. should work even if firstChild is empty.

Reply:
https://www.w3.org/TR/DOM-Level-3-Core/core.html#ID-184E7107